src/main/java/org/gridsuite/filter/server/dto/TwoWindingsTransformerFilter.java

```diff
@@ -58,6 +66,8 @@ public boolean isEmpty() {
             && substationName == null
             && CollectionUtils.isEmpty(countries)
             && nominalVoltage1 == null
-            && nominalVoltage2 == null;
+            && nominalVoltage2 == null
+            && freeProperties == null
```

Comment:
useless, it's checked by CollectionUtils.isEmpty just after ...

Reply:
Good catch too :-)

Reply:
Done